### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by niallmcdonough** [PopGoesTheWza/ts-gas-project-starter#7]
Hi, 
Are there any blog posts or documentation showing how people use this project? I get no errors but Code.js just includes the reference lines when I run build-clean and cannot figure it out. For example, there's no explanation of what FileA.ts and Code.ts in src/GasProject2 are for
Thanks

**Comment by ryancwalsh** [comment]
I agree. I came here from the link at https://github.com/google/clasp/blob/e851215b8abe4de282c62c4d61076c85e89a56ba/docs/typescript.md#typescript-support and have spent so many hours trying to figure out this project, and I still can't get this pattern to work in my own.

I will follow up here: https://github.com/google/clasp/issues/850#issuecomment-1178422476

**Comment by PopGoesTheWza** [comment]
Thanks for your interest in this template.

I am no longer actively using Google Apps Scripts in my projects; hence my late reply.

If there is still interest/need, I will try to refresh the template.

Regarding documentation and my lack of bandwidth, if one can suggest an outline and what needs to be clarified.

**Comment by ryancwalsh** [comment]
Hi @PopGoesTheWza . Thanks for the response.

If you do end up fleshing out the Readme so that it's foolproof for someone to make an Apps Script project in TS, I'll probably end up using it because I do still have an Apps Script project that I use every month or so.

But I'd understand if you have no interest in doing that since you haven't been using this repo in so long.

I've added myself as a watcher of this repo.  Thanks.

P.S. I'm surprised Google wouldn't add official support.
